src/main/java/org/colonelkai/publictransit/config/PublicTransitConfigNodes.java

```diff
@@ -1,25 +1,35 @@
 package org.colonelkai.publictransit.config;
 
+import org.colonelkai.publictransit.PublicTransit;
 import org.colonelkai.publictransit.config.node.*;
 import org.colonelkai.publictransit.line.CostType;
 import org.core.config.ConfigurationNode;
 
 interface PublicTransitConfigNodes {
 
+    // CONFIG FILES
+    final Config MAIN_CONFIG = PublicTransit.getPlugin().getConfig();
```

Comment:
Creating a short throw like this is bad practise as its duplicating the entry point leading to inconsistancies across all code that access your code. 

This may also have a dependency loop due the location if it

Reply:
here u go